### ISSUE TITLE
Improve layout and presentations in detail view

### ISSUE THREAD

**Issue by dennereed** [paleocore/paleocore#98]
The detail view presentation is awkward and needs to be improved.

**Comment by wabarr** [comment]
I agree it is currently ugly, but I am not quite sure how to make it better.  Maybe implement something like fieldsets in a django form, where we would group related fields together?
